### PR TITLE
[IMPROVEMENT] - Adding Stale Bot

### PR DESCRIPTION
**Pull request by sethkfman**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
Uses: https://github.com/apps/stale

Adding stale bot to close GH Issues and PRs with the following rule set:

- Issue/PR will be label stale if no activity has been seen for 90 days
- Issue/PR will be closed 7 days after it has been stale
- Exempt Issues/PRs are:
    - `type-security`
    - `awaiting-metamask`
    - `feature-request`
- When an Issue/PR is label stale there is an message explaining why
- When an Issue/PR is close because it was stale there is a message explaining that is can be reopened
- Before merging add the stale bot GH app

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Review by Cal-L — APPROVED** [review]
I understand configuration may change depending on team's requests, but we can update the configuration as those scenarios come up. LGTM

**Comment by hesterbruikman** [comment]
Great idea! Thanks @sethkfman

**Review by leotm — APPROVED** [review]

File: .github/stale.yml [hunk 1]
```diff
@@ -1,59 +0,0 @@
-# Configuration for probot-stale - https://github.com/probot/stale
```
[the remark this answers is not in the document]
Comment (sethkfman) [reply .github/stale.yml]: Switched to the Security Team suggested GH action. Let me know if you have questions on that configuration.

**Review by legobeat — APPROVED** [review]
one nit, lgtm otherwise

**Pull request merged**